unicodetools/src/main/java/org/unicode/text/UCD/Normalizer.java

```diff
@@ -250,7 +250,9 @@ public void getCompositionStatus(BitSet leading, BitSet trailing, BitSet resulti
     }
 
     public boolean isTrailing(int cp) {
-        return composition ? data.isTrailing(cp) : false;
+        var buffer = new StringBuffer();
+        data.getRecursiveDecomposition(cp, buffer, compatibility);
+        return composition ? data.isTrailing(buffer.codePointAt(0)) : false;
```

Comment:
nit / minor performance
```suggestion
        if (!composition) { return false; }
        var buffer = new StringBuffer();
        data.getRecursiveDecomposition(cp, buffer, compatibility);
        return data.isTrailing(buffer.codePointAt(0));
```